### PR TITLE
Crud

### PR DESCRIPTION
**Pull request by misyo98**

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+class UsersController < ApplicationController
+  def index
+  	@user = User.all
```
Comment (BogdanBusko) [inline app/controllers/users_controller.rb]: 2 spaces everywhere

File: app/models/user.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+class User < ApplicationRecord
+
+	  validates :first_name, :last_name,  presence: true, length: { maximum: 60 }
```
Comment (BogdanBusko) [inline app/models/user.rb]: here to

File: config/routes.rb [hunk 3]
```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
+
+resources :users
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
```
Comment (BogdanBusko) [inline config/routes.rb]: remove comment

File: spec/helpers/users_helper_spec.rb [hunk 4]
```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+# Specs in this file have access to a helper object that includes
```
Comment (BogdanBusko) [inline spec/helpers/users_helper_spec.rb]: remove comment

File: spec/models/user_spec.rb [hunk 5]
```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject {
```
Comment (BogdanBusko) [inline spec/models/user_spec.rb]: where is factory_bot?

File: app/assets/javascripts/users.coffee [hunk 6]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (BogdanBusko) [inline app/assets/javascripts/users.coffee]: Remove files that you don`t use

File: app/controllers/users_controller.rb [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+
+    redirect_to users_path
+  end
+
+  def new
+  	@user = User.new
+  end
+
+  def edit
+  	@user = User.find(params[:id])
```
Comment (pereslop) [inline app/controllers/users_controller.rb]: move this code to the private method (def resource) and reuse it other actions(show/edit/destroy)

File: app/models/user.rb [hunk 8]
```diff
@@ -0,0 +1,18 @@
+class User < ApplicationRecord
+
+  validates :first_name, :last_name,  presence: true, length: { maximum: 60 }
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
```
Comment (pereslop) [inline app/models/user.rb]: add more specs for email, in your specs you check only if email is present, write specs for this regex

File: app/controllers/users_controller.rb [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+
+
+  def destroy
+    @user = User.find(params[:id])
+    @user.destroy
+
+    redirect_to users_path
+  end
+
+    private
```
Comment (pereslop) [inline app/controllers/users_controller.rb]: remove tab

**Pull request closed without merging**